slay/example_workflow/user_package/shared_processor.py

```diff
@@ -0,0 +1,20 @@
+import slay
+
+IMAGE_NUMPY = (
+    slay.Image()
+    .pip_requirements_file(
+        "/home/marius-baseten/workbench/truss/example_workflow/requirements.txt"
```

Comment:
nit: make relative path please

Reply:
I have a TODO for that. It's not that trivial, so I'll do it in a follow up.

Reply:
Actually, I added a "hack" for this with further comments/TODO - it's not very robust, but works in the happy case.